src/distribution/direct/template.sol

```diff
@@ -45,4 +60,9 @@ contract DirectGrantsDistributionStrategy is IDistributionStrategy {
     function setStrategyOwner(address _strategyOwner) public {
         strategyOwner = _strategyOwner;
     }
+
+    // todo: this will be for when we use ERC20 options, not ETH
+    function setToken(address _token) public {
+        token = _token;
+    }
```

Comment:
^ we wouldn't be needing or setStrategyOwner 
These would be set via constructor